OpenRA.Mods.Common/Widgets/ViewportControllerWidget.cs

```diff
@@ -118,7 +118,7 @@ public void UpdateMouseover()
 			}
 
 			var frozen = world.ScreenMap.FrozenActorsAt(world.RenderPlayer, worldRenderer.Viewport.ViewToWorldPx(Viewport.LastMousePos))
-				.Where(a => a.TooltipInfo != null && a.IsValid)
+				.Where(a => a.TooltipInfo != null && a.IsValid && (a.Actor != null && !a.Actor.IsInvisibleTo(world.RenderPlayer)))
```

Comment:
Extra parenthesis are not needed.
Also, a few lines down there is a ```if (frozen.Actor != null)``` check you can remove, because you are now filtering out null actors here.